### ISSUE TITLE
SVGFormatException using rgb values in path element 'style' attribute

### ISSUE THREAD

**Issue by ryanmccauley211** [Unity-Technologies/vector-graphics-samples#10]
When using the style attribute the fill value throws an error for rgb values. Here is the line causing the problem:

```
<path style="stroke:none;fill-rule:nonzero;fill:rgb(0,0,0);fill-opacity:1;" d="M 108 93.046875 L 504 93.046875 L 504 89.0625 L 108 89.0625 Z M 108 93.046875 "/>
```

If I change it to a hex value it works fine. Other elements such as <g> can use rgb values without throwing an exception such as:

```
<g style="fill:rgb(0%,0%,0%);fill-opacity:1;">
...
</g>
```

```
Asset import failed, "Assets/test.svg" > SVGFormatException: SVG Error (line 497, character 2): The given key was not present in the dictionary.
Unity.VectorGraphics.SVGAttribParser.ParseFill (Unity.VectorGraphics.Node node, Unity.VectorGraphics.SVGDictionary dict, Unity.VectorGraphics.SVGStyleResolver styleResolver) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:2024)
Unity.VectorGraphics.SVGDocument.path () (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:502)
Unity.VectorGraphics.SVGDocument.ParseChildren (Unity.VectorGraphics.Node node, System.String nodeName) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:224)
Unity.VectorGraphics.SVGDocument.ParseChildren (Unity.VectorGraphics.Node node, System.String nodeName) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:225)
Unity.VectorGraphics.SVGDocument.svg () (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:988)
Unity.VectorGraphics.SVGDocument.Import () (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:182)
Unity.VectorGraphics.SVGParser.ImportSVG (System.IO.TextReader textReader, Single dpi, Single pixelsPerUnit, Int32 windowWidth, Int32 windowHeight) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:55)
Unity.VectorGraphics.Editor.SVGImporter.OnImportAsset (UnityEditor.Experimental.AssetImporters.AssetImportContext ctx) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Editor/SVGImporter.cs:85)
UnityEditor.Experimental.AssetImporters.ScriptedImporter.GenerateAssetData (UnityEditor.Experimental.AssetImporters.AssetImportContext ctx) (at /Users/builduser/buildslave/unity/build/Editor/Mono/AssetPipeline/ScriptedImporter.cs:19)
```

**Comment by mcoted** [comment]
Our color parsing is a bit lacking at the moment, we will improve it for the next release.

Thank you for reporting this!

**Comment by mcoted** [comment]
This should be fixed with version 1.0.0-preview.19
